launcher/LaunchController.h

```diff
@@ -48,14 +48,16 @@ class LaunchController : public Task {
     void executeTask() override;
 
     LaunchController(QObject* parent = nullptr);
-    virtual ~LaunchController(){};
+    virtual ~LaunchController() {};
```

Comment:
Revert this change to please the format check:
```suggestion
    virtual ~LaunchController(){};
```